x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/action/FollowIndexAction.java

```diff
@@ -222,7 +225,19 @@ public Request() {
 
         @Override
         public ActionRequestValidationException validate() {
-            return null;
+            ActionRequestValidationException validationException = null;
+
+            if (maxRetryDelay.millis() <= 0) {
+                String message = "maxRetryDelay must be positive but was [" + maxRetryDelay.getStringRep() + "]";
```

Comment:
Maybe the message should be centered around the REST parameter `max_retry_delay` as this is the common usage?